server/src/portfoliogrowthRoutes.js

```diff
@@ -0,0 +1,45 @@
+import express from 'express';
+import authMiddleware from './auth/authMiddleware.js';
+import { pgPool } from './db/dbClient.js';
+
+const router = express.Router();
```

Comment:
I think it would be best to define this growth route in the existing portfolioRoutes.js. what do you think?

Reply:
That makes sense. I will move it inside portfolioRoutes.js and call it with router.get('/growth')